dotCMS/src/main/java/com/dotcms/rest/ClusterResource.java

```diff
@@ -196,13 +195,13 @@ public Response getNodesInfo ( @Context HttpServletRequest request, @PathParam (
      * @return
      * @throws DotStateException
      * @throws DotDataException
-     * @throws DotSecurityException
      * @throws JSONException
      */
     @GET
     @Path ("/getNodeStatus/{params:.*}")
     @Produces ("application/json")
-    public Response getNodeInfo ( @Context HttpServletRequest request, @PathParam ("params") String params ) throws DotStateException, DotDataException, DotSecurityException, JSONException {
+    public Response getNodeInfo ( @Context HttpServletRequest request, @PathParam ("params") String params )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Refactor this method to reduce its Cognitive Complexity from 34 to the 15 allowed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3776)